.github/vale/styles/Klaw/branding.yml

```diff
@@ -22,3 +22,5 @@ swap:
   "node[ -]?js?": NodeJS
   "type[ -]?scripts?": TypeScript
   "(?i)ssl": SSL
+  "(?i)klaw": Klaw
+  "(?i)klaw core": Klaw Core
```

Comment:
Since this is now in `branding.yml` (where it belongs), you can remove that line from `branding-warning-temp.yml`